src/katgpucbf/fgpu/engine.py

```diff
@@ -1581,21 +1590,13 @@ def comma_string_to_float(comma_string: str) -> tuple[float, float]:
             phase, phase_rate = comma_string_to_float(phase_str)
 
             delay_samples = delay * self.adc_sample_rate
-            # For compatibility with MeerKAT, the phase given is the net change in
-            # phase for the centre frequency, including delay, and we need to
-            # compensate for the effect of the delay at that frequency. The centre
-            # frequency is 4 samples per cycle, so each sample of delay reduces
-            # phase by pi/2 radians.
```

Comment:
Do we need to maintain the "For compatibility with MeerKAT" component of this comment?
For ease of reference, the comment added on Line 789+ reads
> The phase is referenced to the centre frequency, but coarse delay in wideband affects the 
phase of the centre frequency (each sample shifts it by pi/2) and this needs to be corrected for.

Perhaps also "The centre frequency is 4 samples per cycle" to explain the `np.array(start_coarse_delays) % 4` (?)

Reply:
I don't think we particularly need to state that it's for MeerKAT compatibility; it is defined by the ICD, and a history lesson doesn't really matter. But I'll clarify the 4 samples per cycle and the `% 4`.